src/System.Net.Sockets.Legacy/System.Net.Sockets.Legacy.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
System.Net.Sockets.Legacy is created to separate the Being/End style APM methods of types in System.Net.Sockets (e.g. Socket, TcpClient). We do not want to include these in the forward-looking version of the Sockets contract, but want to provide them for backward compatibility. So we are currently planning to provide a separate contract that will take a dependency on the System.Net.Sockets contract. For more info, see https://github.com/CIPop/corefx/pull/18